main.py

```diff
@@ -35,11 +35,14 @@
     # we do not want the bot to reply to itself
     if message.author == client.user:
         return
-
+     
     # Check if the message author has the Game Master role
-    isGameMaster = False
-    if False: # somebody fix this
+    author = ctx.message.author
```

Comment:
You don't need to put ctx in front of that, it works without that. Also, it is usually not recommended to define variables if you're only gonna use 'em once. You could just write message.author.roles on line 42 instead of author.roles